tests/callbacks/test_stochastic_weight_avg.py

```diff
@@ -46,6 +47,15 @@ def training_step(self, batch, batch_idx):
         def train_dataloader(self):
             return DataLoader(RandomDataset(32, 64), batch_size=2)
 
+    class SWATestModelStep(SwaTestModel):
+        def configure_optimizers(self):
+            optimizer = torch.optim.SGD(self.layer.parameters(), lr=0.1)
+            return {
+                "optimizer": optimizer,
+                "scheduler": torch.optim.lr_scheduler.StepLR(optimizer, step_size=1),
+                "interval": "step",
+            }
+
```

Comment:
Can you update `SwaTestModel` instead to have `__init__(self, batchnorm, interval)`?

So we don't have two separate classes

Reply:
Removed the additional class and added interval as an additional parameter 👍

Reply:
Thanks!